Source/WebKit/UIProcess/Cocoa/WKShareSheet.mm

```diff
@@ -352,7 +352,7 @@ - (void)presentWithParameters:(const WebCore::ShareDataWithParsedURL &)data inRe
 
 - (void)presentWithShareDataArray:(NSArray *)sharingItems inRect:(std::optional<WebCore::FloatRect>)rect
 {
-    WKWebView *webView = _webView.getAutoreleased();
+    RetainPtr webView = _webView.get();
```

Comment:
```suggestion
    RetainPtr webView = _webView;
```

Reply:
This doesn't build. `_webView` is a `WeakObjCPtr`, not a `RetainPtr`.